File: src/test/java/com/tom/student/StudentTest.java
package com.tom.student;

import org.junit.Assert;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    public void highestScoreTest(){
        Student stu= new Student("Tom",60,80);
        Assertions.assertEquals(80,stu.highest());
    }

    @Test
    public void averageTest(){
        Student stu= new Student("Tom",60,80);
        Assertions.assertEquals((60+80)/2,stu.getAverage());
    }
}

File: src/test/kotlin/com/tom/StudentTest.kt
package com.tom.kotlin

import com.kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest{

    @Test
    fun highestTest(){
        val stu= Student("Tom",60,80)
        Assertions.assertEquals(80,stu.highest())
    }

    @Test
    fun averageTest(){
        val stu= Student("Tom",60,80)
        Assertions.assertEquals((60+80)/2,stu.getAverage())
    }

    @Test
    fun gradingTest(){
        val stu= Student("Tom",60,80)
        Assertions.assertEquals('C',stu.grading())
    }
}